src/detail-panels/account-header/account-extra-info.jsx

```diff
@@ -118,6 +120,20 @@ function MultilineFormatted({ text, lineClassName = 'text-multi-line' }) {
       const parts = ln.split(urlRegex).map((part, index) => {
         if (urlRegex.test(part)) {
           return <a key={index} href={part} target="_blank" rel="noopener noreferrer">{part}</a>;
+        } else if (atRegex.test(part)) {
+          return part.split(atRegex).map((subPart, subIndex) => {
+            if (atRegex.test(`@${subPart}`)) {
+              return <a key={subIndex} href={`https://clearsky.app/${subPart}`} rel="noopener noreferrer">@{subPart}</a>;
+            }
+            return subPart;
+          });
+        } else if (emailRegex.test(part)) {
+          return part.split(emailRegex).map((subPart, subIndex) => {
+            if (emailRegex.test(subPart)) {
+              return <a key={subIndex} href={`mailto:${subPart}`} rel="noopener noreferrer">{subPart}</a>;
+            }
+            return subPart;
+          });
```

Comment:
i don't like the increasing complexity where we're getting with nested loops of regex tests tbh. this will probably work for now but any future updates to linkification should probably introduce a 3rd party library to help keep this all maintainable.